dnd-bot/dnd_bot/dc/init.py

```diff
@@ -5,13 +5,15 @@
 
 from dnd_bot.database.database_connection import DatabaseConnection
 from dnd_bot.dc.ui.messager import Messager
+from dnd_bot.database.database_startup_load import MultiverseStartupLoad
 
 activity = nextcord.Activity(name='/help', type=nextcord.ActivityType.listening)
 bot = commands.Bot(command_prefix='$', intents=Intents().all(), activity=activity)
 
 
 @bot.event
 async def on_ready():
+    MultiverseStartupLoad.load_data()
     print('\nBot started successfully')
```

Comment:
Ogólnie to nie wiem czy to jest dobre miejsce, o tym mówiłem wczoraj
![image](https://user-images.githubusercontent.com/81694867/212684515-74557fe5-4214-407d-9336-b4749cb68dc1.png)

Reply:
@kapselccc @Mixss